styles.css

```diff
@@ -616,3 +616,284 @@ section featured speakers
   }
 }
 
+/*
+ ===================================
+****|| DESKTOP VERSION  MEDIA QUERIES ||****
+=================================== */
+@media screen and (min-width: 768px) {
+  .btn-toggle {
+    display: none;
+  }
```

Comment:
- [ ] Kindly follow the styles of template as much as you can 

## I made  this videos to show what you need to correct please take a look at them 

####  https://www.loom.com/share/37ebe5c1cd1f40678db9620f55a1389c
#### https://www.loom.com/share/1fbc538318804d438f4ef4ac26d0013f

Reply:
Thank you @andgarzonmal  for the loom videos,they were of a great help to me